py/torch_tensorrt/dynamo/conversion/impl/unary/base.py

```diff
@@ -9,6 +12,23 @@
 )
 from torch_tensorrt.dynamo._SourceIR import SourceIR
 from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor, TRTUnaryOp
+from typing import Union, Callable, Any, Optional
+
+SUPPORTED_ATEN_TORCH_UNARY_OPS: Dict[
+    TRTUnaryOp, Union[Callable[[Any], Any], Callable[[Any, Any], Any]]
+] = {}
+
+
+def get_python_op_from_trt_unary_op(
```

Comment:
1. @gs-olive will implement a generic schema_validator 
```py
from converter_utils import schema_validator 

@register_dynamo_converter(..., capability_validator=schema_validator(n, ["aten::sqrt(Tensor, float) -> Tensor"])
def sqrt(...):
    ...
```
2. @apbose rebase this pr on main once merged and use schema validator to filter any non tensor op
3. @gs-olive implement the DYNAMO_EVAL registry to handle other cases